R/table_bnc.R

```diff
@@ -7,4 +7,11 @@ table_bnc <- function(dat,
     } else if (tolower(model) == "bam") {
 
     }
-}
\ No newline at end of file
+
+  # REMINDERS: add in code that
```

Comment:
Yeah this fxn along with a couple others are incomplete and out of date. Same formatting from spawning biomass and recruitment apply though